osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/DeviceChannelsHelper.java

```diff
@@ -161,11 +183,34 @@ private short readShort(final List<GetResult> resultList, final int index, final
         return attrAddresses;
     }
 
+    protected ObisCode getObisCode(final int channel) {
+        return new ObisCode(String.format(OBIS_CODE_TEMPLATE, channel));
+    }
+
+    protected MethodResultCode deinstallSlave(final DlmsConnectionManager conn, final DlmsDevice device,
+            final short channel, final CosemObjectAccessor mBusSetup) throws ProtocolAdapterException {
+        // in blue book version 10, the parameter is of type integer
+        DataObject parameter = DataObject.newInteger8Data((byte) 0);
+        conn.getDlmsMessageListener().setDescription("Call slave deinstall method");
+        MethodResultCode slaveDeinstall = mBusSetup.callMethod(MBusClientMethod.SLAVE_DEINSTALL, parameter);
```

Comment:
I would rename this variable to slaveDeinstallResultCode to clearly match what it represents.

Reply:
renamed